### PR TITLE
Add the field GroupID to the Milestone struct

### PR DESCRIPTION
**Pull request by WinXaito**
See issue #1749 

I'm going to test my project with my branch to see if this correction works as expected.

### CODE REVIEW

File: group_boards_test.go [hunk 1]
```diff
@@ -359,6 +359,7 @@ func TestGroupIssueBoardsService_UpdateIssueBoard(t *testing.T) {
 			ID:          44,
 			IID:         1,
 			ProjectID:   0,
+			GroupID:     5,
```
Comment (svanharmelen) [inline group_boards_test.go:362]: Please remove the `ProjectID` line here if this test is for `GroupID`

File: milestones.go [hunk 2]
```diff
@@ -37,6 +37,7 @@ type Milestone struct {
 	ID          int        `json:"id"`
 	IID         int        `json:"iid"`
 	ProjectID   int        `json:"project_id"`
+	GroupID     int        `json:"group_id"` // See issue #1749
```
Comment (svanharmelen) [inline milestones.go:40]:
```suggestion
	GroupID     int        `json:"group_id"`
	ProjectID   int        `json:"project_id"`
```

File: issues_test.go [hunk 3]
```diff
@@ -857,3 +857,79 @@ func TestGetIssueParticipants(t *testing.T) {
...
+
+	want := &Issue{
+		ID:                1,
+		Description:       "This is test project",
+		Author:            &IssueAuthor{ID: 1, Name: "snehal"},
+		Assignees:         []*IssueAssignee{{ID: 1}},
+		MergeRequestCount: 1,
+		Milestone: &Milestone{
+			DueDate:     nil,
+			ProjectID:   0,
```
Comment (svanharmelen) [inline issues_test.go:922]:
```suggestion
```

File: issues_test.go [hunk 4]
```diff
@@ -857,3 +857,79 @@ func TestGetIssueParticipants(t *testing.T) {
...
+	want := &Issue{
+		ID:                1,
+		Description:       "This is test project",
+		Author:            &IssueAuthor{ID: 1, Name: "snehal"},
+		Assignees:         []*IssueAssignee{{ID: 1}},
+		MergeRequestCount: 1,
+		Milestone: &Milestone{
+			DueDate:     nil,
+			ProjectID:   1,
+			GroupID:     0,
```
Comment (svanharmelen) [inline issues_test.go:885]:
```suggestion
```

File: issues_test.go [hunk 5]
```diff
@@ -857,3 +857,79 @@ func TestGetIssueParticipants(t *testing.T) {
 		t.Errorf("Issues.GetIssueParticipants returned %+v, want %+v", issueParticipants, want)
 	}
 }
+
+// See issue #1749 - Issue with a group milestone
```
Comment (svanharmelen) [inline issues_test.go:861]:
```suggestion
```

**Comment by WinXaito** [comment]
I tested the fix with my bot, and that's look good to me.

**Review by svanharmelen — APPROVED** [review]
LGTM 👍🏻  Thanks...

**Pull request merged**